constants.xml

```diff
@@ -204,6 +204,11 @@
         <RESUME_BLACKLIST_DELAY_IN_SECONDS>30</RESUME_BLACKLIST_DELAY_IN_SECONDS>
         <INCRDB_DSNUMS_WITH_STATEDELTAS>10</INCRDB_DSNUMS_WITH_STATEDELTAS>
     </recovery>
+    <server>
+        <IP_TO_BIND>127.0.0.1</IP_TO_BIND>
+        <SERVER_PORT>4201</SERVER_PORT>
```

Comment:
rename `SERVER_PORT` to something suitable if we are changing the name of `SHARD_SERVER`